packages/frontend-api/src/Model/Resolver/Article/ArticleResolver.php

```diff
@@ -54,13 +63,64 @@ public function resolver(string $uuid): Article
         }
     }
 
+    /**
+     * @return \Shopsys\FrameworkBundle\Model\Article\Article
+     */
+    public function termsAndConditionsArticle(): Article
+    {
+        try {
+            $articleId = $this->setting->getForDomain(
+                Setting::TERMS_AND_CONDITIONS_ARTICLE_ID,
+                $this->domain->getId()
+            );
+            return $this->articleFacade->getById($articleId);
+        } catch (ArticleNotFoundException $articleNotFoundException) {
+            throw new UserError($articleNotFoundException->getMessage());
+        }
+    }
+
+    /**
+     * @return \Shopsys\FrameworkBundle\Model\Article\Article
+     */
+    public function privacyPolicyArticle(): Article
+    {
+        try {
+            $articleId = $this->setting->getForDomain(
+                Setting::PRIVACY_POLICY_ARTICLE_ID,
+                $this->domain->getId()
+            );
+            return $this->articleFacade->getById($articleId);
+        } catch (ArticleNotFoundException $articleNotFoundException) {
+            throw new UserError($articleNotFoundException->getMessage());
+        }
+    }
+
+    /**
+     * @return \Shopsys\FrameworkBundle\Model\Article\Article
+     */
+    public function cookiesArticle(): Article
+    {
+        try {
+            $articleId = $this->setting->getForDomain(
+                Setting::COOKIES_ARTICLE_ID,
+                $this->domain->getId()
+            );
+            return $this->articleFacade->getById($articleId);
```

Comment:
CookiesFacade::findCookiesArticleByDomainId() should be used here

Reply:
changedd